backend/wrapped/models.py

```diff
@@ -0,0 +1,76 @@
+from django.db import models
```

Comment:
I know we talked about this before but I think we should actually make Semester a model again.
Reason I'm saying this is for admin page purposes, we can use inline models to manage all the pages for a particular semester at a time which seems like a clean way to do it.

We would make a semester model with 1 field, and make it many to many on pages.

We should think of a Page from now on as a page reusable across semesters

Reply:
Leaving this here because we should probably use this later: https://django-admin-sortable2.readthedocs.io/en/latest/

Reply:
So you have to also make GlobalStatKey model